Lesson 2

ATSchoolENikonov opened the pull request:
1,2,3,5 задания со второго урока

ATSchoolENikonov committed:
нет задания с переменными

ATSchoolENikonov committed:
1,2,3,5 задание

LittleJoAna commented on HW2Lesson.java:
```diff
@@ -0,0 +1,67 @@
+        invertArray();//метод для задания 1
+        fillArray();//метод для задания 2
+        changeArray();//метод для задания 3
+        maxminArray();//метод для задания 5
```
Название метода нужно писать lowerCamelCase

LittleJoAna commented on HW2Lesson.java:
```diff
@@ -0,0 +1,67 @@
+            } else arr[i]--;
+            System.out.printf(arr[i] + " ");
+        }System.out.printf(" \n");
+    }
```
Вынести создание массива в метод main
Работа с консолью происходит в методе main
Это сделает ваш метод универсальным.
Интересное решение, можно оптимизировать до тернарного оператора.

LittleJoAna commented on HW2Lesson.java:
```diff
@@ -0,0 +1,67 @@
+            System.out.print(arr2[j] + " ");
+
+        }System.out.printf(" \n");
+
```
В виде итератора лучше использовать переменную i, так код будет более читабелен. Переменная j используется обычно для внутреннего цикла.
Вынести создание массива, работа с консолью в методе main

LittleJoAna commented on HW2Lesson.java:
```diff
@@ -0,0 +1,67 @@
+            if (arr3[d]<6) {
+                arr3[d]=arr3[d]*2;
+            }System.out.print(arr3[d]+ " ");
+        }System.out.printf(" \n");
```
Вынести создание массива из метода, работа с консолью в методе main. 
Про итератор тоже что и выше. 
В решении можно использовать оператор умножение с присваиванием, оно записывается так *=
Также советую перед выгрузкой домашки наживать в Идее кнопки Ctrl + Alt + L

LittleJoAna commented on HW2Lesson.java:
```diff
@@ -0,0 +1,67 @@
+        }System.out.println("max is "+max);
+        System.out.println("min is "+min);
+    }
+}
```
Вынести создание массива, работа с консолью в методе main
Советую разделить это задание на два метода, тогда, если вам понадобится найти только минимум, вам не придется переписывать метод. Также вычисленные переменные должны быть возвращены в вызывающий метод.

ATSchoolENikonov committed:
1.Вынес создание массива в метод Main(это и есть вынос?)
2.Для первого задания сделал тернарный оператор(приращение ++ и -- в таком случае не работает) оператор
3.переименовал переменные на i
4.Разделил мин и макс на 2 метода

ATSchoolENikonov committed:
задача для 3 урока(работает)
